app/lib/matchmaking.rb

```diff
@@ -0,0 +1,6 @@
+module Matchmaking
+  def config
```

Comment:
Despite my growing distaste for Rails' `config.x.`, that's not why I added this. 😂 I'd prefer to stub our own methods instead of stubbing something from the framework so this lets us to `allow(Matchmaking).to receive(:config).and_return({})` which is a bit more tidy. It adds a layer of indirection but I don't think it's too bad.